### DISCUSSION TITLE
Email package author on build regressions

### DISCUSSION THREAD

**Discussion by wadetregaskis** [SwiftPackageIndex/SwiftPackageIndex-Server#2500] [Ideas]
I’d love to receive emails whenever any of my packages have a regression - for example, a new version of my package stops building on a platform or Swift version that it used to work on.

I do try to use GitHub actions to build and test my packages as well, but frankly GitHub’s infrastructure and capabilities there are inferior to the Swift Package Index’s.

Tangentially:  I’d be quite happy to have this feature behind a modestly-priced subscription, as a way to help support SPI.  I already do support SPI through GitHub donations but to be honest I’d like to see the SPI have a more sustainable and scalable funding model.  It’s already such a critical resource for the Swift community and I don’t want it to go away nor be constrained in what functionality it can offer.

**Comment by daveverwer** [comment]
Again, thank you for your kind words and your support of the project, Wade!

> I do try to use GitHub actions to build and test my packages as well, but frankly GitHub’s infrastructure and capabilities there are inferior to the Swift Package Index’s.

What an incredibly nice thing to say. Thank you!

I think this ties in to what I was just saying in the other thread, there are many useful things we could do if we had some kind of authentication. It's something I'd personally like to bring to the site.
